### DISCUSSION TITLE
The Biological meaning of Reps in Pseudobulk-Replicates

### DISCUSSION THREAD

**Discussion by JerryZhang-1222** [GreenleafLab/ArchR#889] [Questions / Documentation]
Hello! First, thanks very much for providing a wounderful tool for snATACseq analysis! So far it helped me a lot. But I have a question about the biological meanings of the "Reps" in the  Pseudo-bulk Replicates Chapter.

In the manuel script, the 'Example Data Set' shows that there are six samples (A-F) and in the following steps we will get 3 to 5 replicates. What if I have 6 samples and sample A,B are under condition X, sample C,D are under condition Y, and sample EF are under condition Z? 

1.First, if my cells are not enough as "cluster4" ro "cluster5" in the Example Data Set, then I will get 3 replicates whose cells are from both condition X,Y and condition Z, then what is the biological meaning of these three replicates? 
2.Second, if my cells in each samples are as same as "cluster2" in the Example Data Set, then I will get ①Rep1 from Sample A under condition X, ②Rep2 from Sample C under condition Y, ③Rep3 from Sample D under condition Y, **④Rep 4 from both D,E,F.** And I can‘t figure out what is the biological meaning of Rep4.

Maybe Sample A-F have to be parallel experiments under the same condition? If so, what should I do if I have samples under differen conditions?Or did I miss some information in the Manuel Script?

Thank you VERY MUCH!

**Comment by rcorces** [comment]
Your question is too convoluted to follow. The flowchart here describes exactly how replicates are created and I dont have any way of making it more clear. You define the cell groupings (for ex clusters). The rest follows the flow chart.
https://www.archrproject.com/bookdown/how-does-archr-make-pseudo-bulk-replicates.html
